frontend/src/components/ViewActivities/index.js

```diff
@@ -70,6 +68,28 @@ class ViewActivities extends React.Component {
       newAct: null 
     }); 
   };
+      
+  /** @inheritdoc */
+  componentDidMount() {
+    app.firestore()
+        .collection(DB.COLLECTION_TRIPS)
+        .doc(this.tripId)
+        .get()
+        .then(doc => {
+          this.setState({
+            collaborators: doc.get(DB.TRIPS_COLLABORATORS),
```

Comment:
Just a heads up when merging master into this branch, the constant in get() has changed.